GitUpKit/Views/GIQuickViewController.m

```diff
@@ -235,3 +250,212 @@ - (BOOL)diffFilesViewController:(GIDiffFilesViewController*)controller handleKey
 }
 
 @end
+
+#import "GICommitListViewController.h"
+@interface GIQuickViewControllerWithCommitsList () <GICommitListViewControllerDelegate>
+@property (strong, nonatomic, readonly) GICommitListViewController *commitListViewController;
+@property (strong, nonatomic, readonly) GIQuickViewController *quickViewController;
+
+@property (strong, nonatomic, readwrite) NSLayoutConstraint *hiddenConstraint;
+@property (strong, nonatomic, readwrite) NSLayoutConstraint *revealedConstraint;
+@end
+
+@implementation GIQuickViewControllerWithCommitsList
+
+#pragma mark - Accessors
+- (GICommitListViewController *)commitListViewController {
+  return self.childViewControllers.firstObject;
+}
+
+- (GIQuickViewController *)quickViewController {
+  return self.childViewControllers.lastObject;
+}
```

Comment:
These shouldn't be based on the position they were added in child view controllers. Make a direct property that holds a reference to the created view controller.